department-of-veterans-affairs/abd-vro
# MCP-2097 Add summary statistics and sufficiency decision to logging

## What was the problem?

Need more logging to monitor summary statistics and sufficiency decision.
Per requirements, disability claims for increase should only show data from 2 years prior to date of claim.
https://docs.google.com/document/d/1oSmrpquL98KoNzMAKITcUiCIaG4fXTcSmir0VJwsQsI/edit#heading=h.t5m5xdifo2yy

Associated tickets or Slack threads:

- [MCP-2097](https://amida.atlassian.net/browse/MCP-2097)

## How does this fix it?[^1]

Add logging statement. Add two years data filters.

## How to test this PR
- Step 1 Try automatedClaim route and look at docker logs for Python containers.
- Step 2[^secrel]

To Do: https://amida.atlassian.net/browse/MCP-2185

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
